acceptance: Add accepted error to TestUpdate

tbg opened the pull request:
Increasing the timeout caused another error to pop up.
Last attempt to de-flake that test before disabling it for now.

Closes #6715.

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/6724)

<!-- Reviewable:end -->

tamird commented on acceptance/update_test.go:
```diff
@@ -127,7 +133,7 @@ func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig
 
 	// Unfreezing again should be a no-op.
-	if reply, err := postFreeze(c, false); err != nil {
+	if reply, err := postFreeze(c, false, long); err != nil {
```
i think you mixed long and short - wouldn't this no-op be the fast one?

tschottdorf replied:
I want only the command which I'm expecting to time out to actually time out (but I'm not ready to wait a minute). For everything else, I want to reduce spurious errors, so a long timeout.

tamird commented on acceptance/update_test.go:
```diff
@@ -86,7 +90,9 @@ func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig
 	// Note that this is the freeze trigger stalling on the Replica, not the
 	// Store-polling mechanism.
-	if reply, err := postFreeze(c, true); !testutils.IsError(err, "timed out waiting for Range|Timeout exceeded while") {
+	acceptErrs := strings.Join([]string{"timed out waiting for Range",
```
nit this is hard to read. how about
```
strings.Join([]string{
"foo",
"bar",
}, "|")
```

tschottdorf replied:
I see your point, but given that this test fails pretty reliably on master, I don't want to invest another CI period.

tamird replied:
sgtm. 🚢

bdarnell commented:
LGTM

<details><summary><i>Previously, tschottdorf (Tobias Schottdorf) wrote…</i></summary>
> #### acceptance: Add accepted error to TestUpdate
> 
> Increasing the timeout caused another error to pop up.
> 
> Last attempt to de-flake that test before disabling it for now.
> 
> 
> 
> Closes #6715.
> 
> 
> </details>


---

Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/6724#-:-KHvHCI0c3vjxTTMNsrq:676834233)*
<!-- Sent from Reviewable.io -->

tbg committed:
acceptance: Add accepted error

Increasing the timeout caused another error to pop up.
Last attempt to de-flake that test before disabling it for now.

tbg committed:
acceptance: skip TestRaftUpdate

tschottdorf commented:
Skipped the test (though I'm hoping the update I pushed along with that does the trick).
Will run this in loops on my personal fork and open a new PR when I'm pretty confident it's not flaky any more.
